backend/src/helpers/mailer.ts

```diff
@@ -0,0 +1,57 @@
+const nodemailer = require('nodemailer');
+require('dotenv').config();
+import { ErrorHandler } from './error';
+import { validEmail } from '../utils/validators';
+
+const createTransporter = async (): Promise<null | ReturnType<
+  typeof nodemailer.createTransport
+>> => {
+  const transporter = nodemailer.createTransport({
+    pool: true,
+    host: 'smtp.gmail.com',
+    port: 465,
+    secure: true,
+    auth: {
+      type: 'OAuth2',
+      user: process.env.SENDER,
+      serviceClient: process.env.EMAILSERVICECLIENT,
+      privateKey: process.env.EMAILPRIVATEKEY,
+    },
+  });
+  try {
+    await transporter.verify();
+    return transporter;
+  } catch (err) {
+    throw new ErrorHandler(500, { status: 'Mailer Error' });
+  }
+};
+
+const checkEmails = (emailadressses: string[]) => {
+  emailadressses.forEach(email => {
+    if (!validEmail(email)) {
+      throw new ErrorHandler(400, { status: 'Validation Error' });
+    }
+  });
+};
+
+export const sendMail = async (
+  list: Array<string>,
+  replier: string,
+  subject: string,
+  email: string,
+) => {
+  checkEmails([...list, replier]);
+  const mailOptions = {
+    from: process.env.SENDER,
+    to: list.join(),
```

Comment:
Receivers should be in the BCC field. If `to` has to be non-empty either set the sender as receiver also, or send to `bedriftskontakt` (Maybe not during development)

Reply:
Fixed 👍

This made me realize that we might have to do something about the fact that our mail might be flagged as spam, if we send too much mail programatically....  Our online.ntnu.no domain is not trusted by outside sources so that might get us flagged. 

I guess it should be fine for now, and for this use case, but we should be aware that we might need to implement a second service that holds the email account for us, and that is trusted so we lessen the likelihood of getting marked as spammers if we scale up this system to provide more services. 

If we need to, i'll suggest [sendGrid](https://sendgrid.com/).

Reply:
True, We have actually had that problem before. 

That can be a solution.

Preferably in the long run, the system will rely on in-app communication rather than emails.